engine/src/test/java/com/google/android/fhir/sync/SyncJobTest.kt

```diff
@@ -198,4 +199,53 @@ class SyncJobTest {
 
     job.cancel()
   }
+
+  @Test
+  fun `sync time should update on every sync call`() = runBlockingTest {
```

Comment:
```suggestion
  fun `Sync time should update on every sync call`() = runBlockingTest {
```

and if you can fix the rest of the test cases in this file it'd be great.

Reply:
Done

Reply:
@jingtang10 As discussed reverted this change